app/javascript/controllers/index.js

```diff
@@ -42,3 +45,4 @@ application.register('select-all', SelectAllController)
 application.register('sidebar', SidebarController)
 application.register('sidebar-group', SidebarGroupController)
 application.register('truncated-text', TruncatedTextController)
+application.register('nested-form', NestedForm)
```

Comment:
I wasn't able to access the nested form controller until I added this - not sure if there is a more standard way to do this. I realize the rest of the imports in here are all local

Reply:
The `court_order_form_controller.js` uses it, too. But it loads it directly. This is the right thing to do for your use case.